ambry-server/src/test/java/com.github.ambry.server/StatsManagerTest.java

```diff
@@ -260,12 +260,13 @@ private StatsSnapshot generateRandomSnapshot() {
    * Mocked {@link StorageManager} that is intended to have only the overwritten methods to be called and return
    * predefined values.
    */
-  private class MockStorageManager extends StorageManager {
+  private static class MockStorageManager extends StorageManager {
+    private static final VerifiableProperties PROPS = new VerifiableProperties(new Properties());
```

Comment:
because Properties is mutable, I don't really want to make that global. But I could add an EMPTY_VPROPS to TestUtils.